src/EnergyPlus/WaterThermalTanks.cc

```diff
@@ -8623,22 +8674,28 @@ namespace WaterThermalTanks {
         using DataGlobals::SecInHour;
         using DataGlobals::WarmupFlag;
         using DataHeatBalance::HeatReclaimDXCoil;
+        using DataHeatBalance::HeatReclaimSimple_WAHPCoil;
         using DataHVACGlobals::ShortenTimeStepSys;
+        using DataGlobals::HourOfDay;
+        using DataGlobals::TimeStep;
+        using DataGlobals::TimeStepZone;
         using DataHVACGlobals::TimeStepSys;
+        using DataHVACGlobals::SysTimeElapsed;
         using DataLoopNode::Node;
         using DXCoils::DXCoil;
         using General::RoundSigDigits;
         using General::SolveRoot;
         using Psychrometrics::CPHW;
         using Psychrometrics::RhoH2O;
         using ScheduleManager::GetCurrentScheduleValue;
+        using WaterToAirHeatPumpSimple::SimpleWatertoAirHP;
 
         // Locals
         // SUBROUTINE ARGUMENT DEFINITIONS:
 
         // SUBROUTINE PARAMETER DEFINITIONS:
         int const MaxIte(500);     // Maximum number of iterations for RegulaFalsi
-        Real64 const Acc(0.00001); // Accuracy of result from RegulaFalsi
+        Real64 const Acc(0.001); // Accuracy of result from RegulaFalsi
```

Comment:
What justifies this change in convergence criterion?

Reply:
I separated the use cases of this ``Acc`` variable, and only changed the convergence criterion for my added capability: desuperheater +stratified tank. The stratified tank calculation function varied pretty much from mixed tank calculation function, which made the SolveRoot convergence harder to achieve a high accuracy. I used the same SolveRoot accuracy as function ``CalcHeatPumpWaterHeater`` and it finally worked out well giving no convergence warning message running my new feature model.